### DISCUSSION TITLE
18 gennaio 2022 B - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#7] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/262e6466-5c4e-4226-b4d7-083c3dbcd1de)

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-15 09:53:19](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/390d6c9a-b326-4aaf-944b-2060912e5f8a)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/20c4bb4c-0ae7-4603-be1f-c8276a6bc993)
